lib/compat/wordpress-6.2/block-editor-settings.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Adds settings to the block editor.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Adds styles and __experimentalFeatures to the block editor settings.
+ *
+ * @param array $settings Existing block editor settings.
+ *
+ * @return array New block editor settings.
+ */
+function gutenberg_get_block_editor_settings_6_2( $settings ) {
+	if (
+		defined( 'REST_REQUEST' ) &&
+		REST_REQUEST &&
+		isset( $_GET['context'] ) &&
+		'mobile' === $_GET['context']
+	) {
+		return;
```

Comment:
```suggestion
		return $settings;
```

https://github.com/WordPress/gutenberg/pull/46141/commits/2facb03966774ad40427d27f3eda3d895451b8ed introduced this incorrect return.